src/fusion_power_plant.h

```diff
@@ -259,6 +259,9 @@ class FusionPowerPlant : public cyclus::Facility  {
   double blanket_turnover;
   double fuel_usage_mass;
   static const double burn_rate; // kg/GW-y
+  double startup_inventory;
+  static const double tritium_yearly_decay_fraction;
```

Comment:
From a readability point of view (and possibly a minor, in the weeds, compilation benefit) it probably makes sense to put the `static const` variables together and as the first (or last??) variables in the class.